### PR TITLE
Bug/No request message shown in chat if request from wallet (#2162)

### PR DESCRIPTION
**Pull request by flexsurfer**
[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

addresses #2162

status: ready

### CODE REVIEW

File: src/status_im/chat/handlers/webview_bridge.cljs [hunk 1]
```diff
@@ -1,129 +1,33 @@
 (ns status-im.chat.handlers.webview-bridge
-  (:require [re-frame.core :refer [after dispatch enrich]]
-            [status-im.utils.handlers :refer [register-handler]]
-            [status-im.utils.handlers :as u]
-            [status-im.utils.types :as t]
-            [status-im.i18n :refer [label]]
+  (:require [re-frame.core :refer [dispatch]]
```
Comment (flexsurfer) [inline src/status_im/chat/handlers/webview_bridge.cljs:2]: removed all related to old dapp wallet

File: src/status_im/ui/screens/contacts/navigation.cljs [hunk 2]
```diff
@@ -7,4 +7,10 @@
       (assoc-in [:toolbar-search :show] nil)
       (assoc-in [:contacts/list-ui-props :edit?] false)
       (assoc-in [:contacts/ui-props :edit?] false)
-      (assoc :contacts/click-handler click-handler)))
\ No newline at end of file
+      (assoc :contacts/click-handler click-handler)))
+
+(defmethod nav/preload-data! :contact-list-modal
```
Comment (flexsurfer) [inline src/status_im/ui/screens/contacts/navigation.cljs:12]: just moved to proper ns

File: src/status_im/ui/screens/wallet/choose_recipient/views.cljs [hunk 3]
```diff
@@ -84,10 +84,10 @@
                                                         {:width  (.-width layout)
                                                          :height (.-height layout)}]))}
       (when (or platform/android?
-                camera-permitted?)[camera/camera {:style         styles/preview
+                camera-permitted?)[camera/camera {:style         styles/preview}]
```
Comment (flexsurfer) [inline src/status_im/ui/screens/wallet/choose_recipient/views.cljs]: damn Daniel

**Review by oskarth — APPROVED** [review]

File: src/status_im/ui/screens/wallet/choose_recipient/views.cljs [hunk 4]
```diff
@@ -84,10 +84,11 @@
                                                         {:width  (.-width layout)
                                                          :height (.-height layout)}]))}
       (when (or platform/android?
-                camera-permitted?)[camera/camera {:style         styles/preview
-                      :aspect        :fill
-                      :captureAudio  false
-                      :torchMode (camera/set-torch camera-flashlight)
-                      :onBarCodeRead #(re-frame/dispatch [:choose-recipient (camera/get-qr-code-data %) nil])}])
+                camera-permitted?)
+        [camera/camera {:style         styles/preview
```
Comment (flexsurfer) [inline src/status_im/ui/screens/wallet/choose_recipient/views.cljs:88]: accidentally fixed styles

**Review by jeluard — APPROVED** [review]

**Pull request merged**